src/snowcli/cli/common/flags.py

```diff
@@ -26,6 +26,15 @@
     rich_help_panel=_CONNECTION_SECTION,
 )
 
+TemporaryConnectionOption = typer.Option(
+    False,
+    "--temporary-connection",
```

Comment:
It would be nice to have also a shorter option, wdyt? For example `-x` shouldn't be common

Reply:
Done

Reply:
We agreed that `-x` option is ok and we'll create a task for adding automatic tests finding conflicts flags.